app/services/art_service/reports/pepfar/viral_load_coverage2.rb

```diff
@@ -176,13 +176,46 @@ def breast_feeding(patient_list)
         # rubocop:enable Metrics/MethodLength
 
         def build_report(report)
+          refresh_outcomes_table
+          load_tx_curr_into_report(report, create_patients_alive_and_on_art_query)
           clients = process_due_people
           clients.each do |patient|
             report[patient['age_group']][patient['gender'].to_sym][:due_for_vl] << patient['patient_id']
           end
           load_patient_tests_into_report(report, clients.map { |patient| patient['patient_id'] })
         end
 
+        def refresh_outcomes_table
+          CohortBuilder.new(outcomes_definition: 'pepfar')
+                       .init_temporary_tables(@start_date, @end_date, nil)
+        end
+
+        def create_patients_alive_and_on_art_query
+          ActiveRecord::Base.connection.select_all(
```

Comment:
ActiveRecord::Base.connection.select_all <<~SQL
  your query comes here... 
SQL

no need for the parenthesis here.